src/main/java/seedu/address/model/FriendlyLink.java

```diff
@@ -217,6 +217,18 @@ public void addPair(Pair pair) {
         pairs.add(pair);
     }
 
+    /**
+     * Adds a pair consisting of elderly with {@code elderlyNric} and volunteer with {@code volunteerNric}
+     * from {@code FriendlyLink}.
```

Comment:
Small typo.
```suggestion
     * to {@code FriendlyLink}.
```

Reply:
Resolved in commit 71e90f4f6ec2f74d637d12a4c6f48d26cb9ecb87